unittests/files/opt/seagate/sspl/setup/resource_map/test_storage.py

```diff
@@ -128,8 +128,173 @@
         ]
     }}
     """
+
 ENCLOSURE_RESPONSE_EMPTY = {}
 
+ENCLOSURE_RESPONSE = """
+    {
+    "status_code": 200,
+    "api-response": {
+            "enclosures": [
```

Comment:
@shriya-deshmukh, adding more api responses further will be difficult to manage in this file itself. Can we have all API responses separated from this file and can be imported or loaded?
cc: @thavanathan

Reply:
Please ignore if not possible.
But controller api response is having more arrtibutes. Thats why asking.

Reply:
Yes agree, created separate file for api responses.

Reply:
Great.